aeroprom.conf.dev

```diff
@@ -18,6 +18,21 @@ timeout = 10
 # debug level can be info|warning,warn|error,err|debug
 log_level = "debug"
 
+# Basic HTTP authentication for '/metrics'.
+basic_auth_username=""
+basic_auth_password=""
+
+# Metrics Whitelist - If specified, only these metrics will be scraped. An empty list will exclude all metrics.
+
+# Namespace metrics whitelist
```

Comment:
I think the experience is better if there is just one metric whitelist. The program will filter the ones for each sub-category.
Also, since this list will be long, it should be in the bottom of the config file to avoid hiding other config keys.

Reply:
The set statistics like `tombstones`, `objects`, `truncate_lut` etc. overlap with namespace statistics. That's the reason for keeping individual whitelists for namespace, sets and node metrics.

Good point, I'll move the whitelist config keys to the bottom and it also makes sense to keep them under `[Aerospike]` table.